web/app/js/components/util/Utils.test.js

```diff
@@ -124,4 +125,18 @@ describe('Utils', () => {
       expect(toClassName('test/potato-e1af21-f3f3')).toEqual('test_potato_e_1_af_21_f_3_f_3');
     });
   });
+
+  describe('regexFilterString', () => {
+    it('converts string to a valid regex when filtering namespace selection options', () => {
+      expect(regexFilterString('emojivoto')).toEqual(new RegExp(/emojivoto/));
+      expect(regexFilterString('emojivoto123')).toEqual(new RegExp(/emojivoto123/));
+      expect(regexFilterString('emojivoto*')).toEqual(new RegExp(/emojivoto.+/));
+      expect(regexFilterString('emojivoto**')).toEqual(new RegExp(/emojivoto.+.+/));
+      expect(regexFilterString('Emojivoto')).toEqual(new RegExp(/emojivoto/));
+      expect(regexFilterString('emojivoto{}')).toEqual(new RegExp(/emojivoto/));
+      expect(regexFilterString('emojivoto_.')).toEqual(new RegExp(/emojivoto_./));
+      expect(regexFilterString('emojivoto_.{')).toEqual(new RegExp(/emojivoto_./));
+      expect(regexFilterString('emojivoto//')).toEqual(new RegExp(/emojivoto\/\//));
```

Comment:
while we're here may as well add a few more to test the `/gi` part:
```suggestion
      expect(regexFilterString('emojivoto//')).toEqual(new RegExp(/emojivoto\/\//));
      expect(regexFilterString('emojivoto-prod-1')).toEqual(new RegExp(/emojivoto-prod-1/));
      expect(regexFilterString('emoji??Voto-pr##od-1')).toEqual(new RegExp(/emojivoto-prod-1/));
```